src/app/routes/revisions/revisions.component.html

```diff
@@ -1,13 +1,24 @@
 <div class="container">
   <div class="row">
-    <page class="col" requestURL="{{revisionURL}}"></page>
-    <div class="col">
-      <div *ngFor="let revision of revisions">
-        <i *ngIf="(revision.id == loadedID) || (revision.current && currentSelected)" class="fas fa-arrow-right"></i>
-        <button *ngIf="!revision.current" (click)="loadRevision(revision.id)">{{getDateTime(revision.last_updated)}}</button>
-        <button *ngIf="revision.current" (click)="loadRevision(null)">Current</button>
-      </div>
+    <div class="col-sm-12 col-md-6 offset-md-3 text-center">
+      <h1>Revisions</h1>
+    </div>
+  </div>
+  <div class="row">
+    <page class="col-9" requestURL="{{revisionURL}}"></page>
+    <div class="col-3 list-group">
+      <button [ngStyle]="getStyle(currentRevisionID)" (click)="loadRevision(currentRevisionID)" class="list-group-item list-group-item-action">
+        Current
+      </button>
+      <button [ngStyle]="getStyle(revision.id)" *ngFor="let revision of revisions" (click)="loadRevision(revision.id)" class="list-group-item list-group-item-action">
+        {{getDateTime(revision.last_updated)}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12 w-100">
```

Comment:
I think it might be a little bit more intuitive if these two buttons are turned into a button group that is above or below the list-group of revisions